ui/app/AppLayouts/Profile/controls/InformationTile.qml

```diff
@@ -0,0 +1,53 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.13
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Rectangle {
+    id: root
+
+    property alias primaryText: primaryText.text
+    property alias secondaryText: secondaryText.text
+    property alias tagsModel: tags.model
+    property alias tagsDelegate: tags.delegate
+
+    height: 52
+    width: layout.width + Style.current.xlPadding
+    radius: Style.current.radius
+    border.width: 1
+    border.color: Theme.palette.baseColor2
+    color: Style.current.transparent
+
+    ColumnLayout {
+        id: layout
+        spacing: -1
```

Comment:
Why negative spacing?

Reply:
without it the spacing is higher than we see in the design
<img width="558" alt="image" src="https://user-images.githubusercontent.com/60327365/180211002-1f235229-5447-4bc7-acb0-03fc10061d96.png">